Formula/litani.rb

```diff
@@ -0,0 +1,59 @@
+class Litani < Formula
+  desc "Metabuild system"
+  homepage "https://awslabs.github.io/aws-build-accumulator/"
+  url "https://github.com/ronakfof/aws-build-accumulator/archive/refs/tags/1.19.0.tar.gz"
+  sha256 "7ace2833db4ccf52db21589b4e91d2e5cffae778eecbdfb12998a204032a8225"
+  license "Apache-2.0"
+
+  depends_on "coreutils" => :build
+  depends_on "mandoc" => :build
+  depends_on "scdoc" => :build
+  depends_on "gnuplot"
+  depends_on "ninja"
```

Comment:
This is needed at runtime?

Reply:
Both `mandoc` and `scdoc` are just required to run these 2 commands in the formula - 
```
    system "cd #{bin}/doc && #{Formula["python@3.9"].opt_bin/"python3"} configure"
    system "cd #{bin}/doc && ninja"
```

Reply:
I was asking about `ninja`, since it is clear that `mandoc` and `scdoc` are build-only.

Reply:
Yes, `ninja` is needed at runtime.